src/providers/tari_universe/provider.ts

```diff
@@ -0,0 +1,162 @@
+import { TariPermissions } from "../wallet_daemon/tari_permissions";
+import {
+  SubmitTransactionRequest,
+  TransactionResult,
+  TransactionStatus,
+  SubmitTransactionResponse,
+  Account,
+  TariUniverseProviderRequest,
+  Substate,
+  TemplateDefinition,
+  VaultBalances,
+} from "../types";
+import { TariProvider } from "../index";
+import { Instruction, TransactionSubmitRequest, AccountsGetBalancesResponse } from "@tariproject/wallet_jrpc_client";
+
+export const Unsupported = "UNSUPPORTED";
```

Comment:
Where is it used? As I see the same const is defined in `wallet_daemon/provider.ts` but also not used. Remove it or create new file with constants to not duplicate this one